src/shared_gui_components/EditView.cpp

```diff
@@ -400,4 +402,80 @@ void InputCheckBox::setIncomplete(bool incomplete) {
   }
 }
 
+PathInputView::PathInputView(const std::string& extension, bool isRead)
+  : InputView(), lineEdit(new QLineEdit()), nameLabel(new QLabel()), m_isRead(isRead) {
+  auto* vLayout = new QVBoxLayout();
+  vLayout->setContentsMargins(0, 0, 0, 0);
+  vLayout->setSpacing(5);
+  setLayout(vLayout);
+
+  nameLabel->setOpenExternalLinks(true);
+  nameLabel->setWordWrap(true);
+  vLayout->addWidget(nameLabel);
+
```

Comment:
Does the `lineEdit` need to be connected to `emit selectedPathChanged(toPath(selectedPath))`?  Does that support free text entry or is it read only?

Reply:
Already done, see https://github.com/openstudiocoalition/OpenStudioApplication/pull/761#pullrequestreview-2421146641